### DISCUSSION TITLE
New Modes and Grenades

### DISCUSSION THREAD

**Discussion by Hashimk2** [HasangerGames/suroi#391] [Ideas]
50v50 should be added because it's fun and chaotic and suroi.io will eventually get boring if there is only one mode. With a lieutenant, healer, leader, and lone survivors, it would be more challenging and fun for both teams. If you add MIRV grenades and strobes, it will be more chaos and fun with lots of people.

**Comment by emeraldn30** [comment]
we will eventually add more modes, but we won't copy surviv's 50v50 directly as you suggested.
but 50v50 wouldn't be feasible with the lower playercounts that servers receive most of the time.

**Reply by Hashimk2** [reply]
maybe for only north America

**Reply by ghost** [reply]
We should create a mode that has teams that is more than squads

**Comment by Hashimk2** [comment]
How about  something similar to the potato event

**Comment by AshMyBoi** [comment]
The game is still Beta, We won't copy surviv.io's 50v50 directly as you suggested But yes we will make more Game Modes in the future.

**Reply by ghost** [reply]
Was Surviv.io Kongregate?

**Comment by ghost** [comment]
Whenever I place a C4 in the normal mode, it just vanishes when someone shoots it or hits it. Can we change it so it does not break or else there is really no point of it

**Reply by Hashimk2** [reply]
?

**Reply by Hashimk2** [reply]
> No

No to what?
